docs/roadmap.rst

```diff
@@ -1,70 +1,127 @@
     "shelled", 3dSHORE will be used as the diffusion model. If the data are
     single-shell, we will use SFM as the diffusion model.
 
-#. Eddy current correction.
+#. :math:`B_1` inhomogeneity correction
 
-    We will explore the possible adaptations of the HMC based on SHOREline above.
-    In cases where the data are "shelled", 3dSHORE will be
-    used as the diffusion model. If the data are single-shell, we will use SFM
-    as the diffusion model.
+    - Decide whether it can be brought around from estimation on T1w images
+    - Decide whether it should be a default-off option that can be enabled with
+      a flag, or else, generate both conversions always.
 
-#. Framewise-displacement calculation
+#. Initiate Phase I of testing
 
-    We will identify volumes that are outliers in terms of head motion, or other
-    severe artifacts that make them likely candidates for exclusion from further
-    analysis.
+    - Compose our test-bed dataset
+    - Document Phase I testing and reporting protocols
+    - Start execution
 
-Version 0.5 (May 1st, 2020)
-----------------------------
-#. Registration between dMRI and T1w image.
+# Continue with the development of ISBI 2021 tutorial
+
+Version 0.7 (March, 2021)
+-------------------------
+The *noisy month*. This is not a musical event, but a development cycle where we will
+focus on the implementation of steps addressing noise in DWI:
 
 #. Identification of outlier measurements (+ imputation?)
 
-If we get around to doing thesee steps earlier, they can also be included in
-earlier releases.
+#. Implementation of component-based noise identification techniques
 
+#. Gibbs-ringing: investigate whether it should be estimated if other techniques
+   are in place (i.e., component-based above), and ordering of steps.
 
-Version 1.0 (Targetted for September 2020)
-------------------------------------------
-After integrating the above steps, we will spend the time leading to a 1.0
-testing the software on various datasets, evaluating and validating the
-resulting derivatives.
+#. Rician bias modeling.
+
+#. DWI carpet-plot and confounds collation.
 
+#. Testing Phase I execution
+
+#. Final release of the ISBI 2021 tutorial
+
+Version 0.8 (April, 2021)
+-------------------------
+This release will only address bugfixes conducive to finishing evaluation Phase I,
+which should conform a pretty solid ensemble ready for premiere in ISMRM 2021.
+
+Version 0.9 (May, 2021)
+-----------------------
+#. First official presentation at ISMRM 2021 (should the abstract be accepted)
+#. Evaluation Phase II starts.
+
+  - Determine an appropriate dataset
+  - Plan for benchmarking experiments (`#121 <https://github.com/nipreps/dmriprep/issues/121>`__)
+  - Start with addressing issues as they are reported
+
+Version 1.0 (Targetted for September 2021)
+------------------------------------------
+Wrap-up evaluation Phase II with the first stable release of *dMRIPrep*.
 
```

Comment:
I think it might be helpful to clarify that dmriprep is both dataset-adaptive (i.e. to heterogeneous acquisition/encoding schemes, available metadata) and user-configurable (i.e. providing the option to run denoising or not if associated compute cost would be too high, susceptibility distortion correction or not if the available data doesn't support it).

Reply:
> user-configurable (i.e. providing the option to run denoising or not if associated compute cost would be too high, susceptibility distortion correction or not if the available data doesn't support it).

I would prefer it were not very configurable - and that requires a lot of solid heuristics. But the less human intervention, the better.

Reply:
Fair enough. 

Though I think at least the former (i.e. dataset adaptive) is a key selling point since there are good reasons to think that the same steps are not optimal for all types of dMRI data...